content/source/docs/cloud/run/index.html.md

```diff
@@ -73,6 +73,19 @@ There are three ways to run speculative plans:
 - With the [remote backend](/docs/backends/types/remote.html) configured, running `terraform plan` on the command line starts a speculative plan. The plan output streams to the terminal, and a link to the plan is also included.
 - The runs API creates speculative plans whenever the specified configuration version is marked as speculative. See [the `configuration-versions` API](../api/configuration-versions.html#create-a-configuration-version) for more information.
 
+#### Retry a speculative plan in the UI
+
+Sometimes a speculative plan might fail due to an external factor, and you just need to try to run it again.  Retrying a speculative plan is allowed under the following conditions:
+
+- The user viewing the run has [workspace level permissions](https://www.terraform.io/docs/cloud/users-teams-organizations/permissions.html#workspace-permissions) to be able to queue plans.
+- The Plan itself has failed, through an error or being canceled.
+
+When these conditions are met, you will see a "Retry Run" button as shown below.
+
+![Screenshot: Clicking on the "Retry Run" button to trigger a new run](./images/retry.gif)
+
+Retrying the run will create a new run with the same configuration version. If it is a VCS-backed workspace, the pull request interface will receive the status of the new run, along with a link to the new run.
```

Comment:
```suggestion
If a speculative plan fails due to an external factor, you can run it again using the "Retry Run" button on its page:

![Screenshot: Clicking on the "Retry Run" button to trigger a new run](./images/retry.gif)

Retrying a plan requires permission to queue plans for that workspace. ([More about permissions.](../users-teams-organizations/permissions.html)) Only failed or canceled plans can be retried.

[permissions-citation]: #intentionally-unused---keep-for-maintainers

Retrying the run will create a new run with the same configuration version. If it is a VCS-backed workspace, the pull request interface will receive the status of the new run, along with a link to the new run.
```

- Changing order of info so readers can bail out earlier if they feel like it. 
- Making shorter.
- Formatting info about permissions to match the way we do it elsewhere in TFC docs.
    - The weird link reference to nowhere is so we can do mass updates easier, if we ever change the permissions model in a major way (like we did that one time). 
    - No need to refer to workspace-level perms, especially since there are org-level perms that should also allow this. (Right?)
- No proper-noun caps on "plan."